node/typings/Search.ts

```diff
@@ -107,3 +99,93 @@ interface Breadcrumb {
   href: string
   name: string
 }
+
+interface BiggySearchProduct {
+  name: string
+  id: string
+  timestamp: number
+  product: string
+  description: string
+  reference: string
+  url: string
+  link: string
+  oldPrice: number
+  price: number
+  brand: string
+  brandId: string
+  installment: BiggyInstallment
+  measurementUnit: string
+  unitMultiplier: number
+  tax: number
+
```

Comment:
there are some blank lines within the types, can we remove them?

Reply:
Done 👍